e2e-tests/discovery/run-test.sh

```diff
@@ -50,18 +50,24 @@ echo "---------------------------------------"
 echo "Registering care organization on Discovery Service..."
 echo "---------------------------------------"
 curl --insecure -s -X POST http://localhost:21323/internal/discovery/v1/dev:eOverdracht2023/${DID}
+# Start Discovery Server
+docker compose up --wait nodeA
+# Registration refresh interval is 500ms, wait some to make sure the registration is refreshed
+sleep 2
 
 echo "---------------------------------------"
-echo "Restarting to force registration on Discovery Service..."
+echo "Searching for care organization registration on Discovery Server..."
 echo "---------------------------------------"
-docker compose up --wait nodeA
-docker compose down nodeB
-docker compose up --wait nodeB
+RESPONSE=$(curl --insecure "http://localhost:11323/internal/discovery/v1/dev:eOverdracht2023?credentialSubject.organization.name=Care*")
+echo $RESPONSE
```

Comment:
there's no assert just an echo, so nothing is tested.

Reply:
Oops forgot